aiogram_dialog/widgets/kbd/scrolling_group.py

```diff
@@ -50,13 +50,15 @@ async def process_callback(self, c: CallbackQuery, dialog: Dialog, manager: Dial
         if data == PAGE_NEXT:
             await self.set_page(c, page + 1, manager)
         elif data == PAGE_PREV:
-            await self.set_page(c, page - 1, manager)
+            await self.set_page(
+                c, page - 1, manager
+            ) if page > 0 else await self.set_page(c, 0, manager)
```

Comment:
incodrrect usage of ternary operator.

Reply:
Could you explain why? or you basically don't use it. I can refactor without it.

Reply:
ternary operator is normally used to select one value or another depending on condition. Here you use it  to control flow of program. For this case is better to use simple if else statements, not ternary.

Also, in this case it is simplier to write ` await self.set_page(c, max(0, page - 1), manager)`.
But as i understand we have the same problem with next-page button and it cannot be fixed so simply

Reply:
> ternary operator is normally used to select one value or another depending on condition. Here you use it to control flow of program. For this case is better to use simple if else statements, not ternary.
Thanks, i got it.
> await self.set_page(c, max(0, page - 1), manager)
Anyway this is more elegant solution.
> But as i understand we have the same problem with next-page button and it cannot be fixed so simply
No, there is not. The issue only with previous-page button. 
But there is a issue with `None` type of `self.widget_id`. In this case [function](https://github.com/Tishka17/aiogram_dialog/blob/develop/aiogram_dialog/widgets/kbd/scrolling_group.py#L60) will not return all data, because page number always 0.